doc/nrf/ug_thread_configuring.rst

```diff
@@ -232,6 +232,20 @@ Minimal Thread Device (MTD)
   By default, the MTD operates as Minimal End Device (MED).
   To make it operate as Sleepy End Device (SED), set :kconfig:`CONFIG_OPENTHREAD_MTD_SED`.
 
+.. _thread_ug_tfm_support:
+
+Trusted Firmware-M support options
+==================================
+
+You can configure your Thread application to run with Trusted Firmware-M by enabling the following Kconfig options:
+
+* :kconfig:`CONFIG_BUILD_WITH_TFM`
+* :kconfig:`CONFIG_OPENTHREAD_CRYPTO_PSA`
+
+These options are enabled by default for the :ref:`application core <ug_nrf5340_intro_app_core>` of the ``nrf5340dk_nrf5340_cpuapp_ns`` build target, which is supported by most of the :ref:`openthread_samples` in the |NCS|.
```

Comment:
If "which" refers to the build target, it should maybe be "that" instead (without the comma, of course). Or you could split the long sentence -> ...build target. It is supported by most of the...